v4.5/Makefile

```diff
@@ -0,0 +1,322 @@
+PROFILE := v4.5
```

Comment:
I would very much prefer we __do__ add 3 sets for 4.5, after all that's the point of the profiles. Our approach of pointing the different profiles to the same assemblies is a hack that should go away.